content/blog/standup.md

```diff
@@ -0,0 +1,212 @@
+
+You can be creative as soon as meetings are short and effective.
+
+##### Q: Do I need a burn down chart? Or any chart? any board?
+
+That is a great way to visualize the status. However, status is not the goal; action item is the goal.
+Anything that helps the project.
+It helps identify whether you need to organize yourself and the team better.
+
+Another way is to have just sticky notes or cards in a dashboard.
+Identify how many cards should have been moved and current status.
+
+Any tool that publishes data is good enough.
+
+Many teams don't even use any! (These are very mature teams. I would recommend having some visual representation.)
+
+### What not to do
+
+##### Too granular
+
+"I worked on the order class and implemented that method", "I worked on liquibase scripts", nobody cares and should care about that.
+Make it abstract enough as if the recipients are non devs.
+The meeting is for the project, not yourself. Don't go too technical.
+
+Focus on the task or story's completeness or challenge.
+
+This is one of the reasons the meeting becomes boring.
+Because nobody understands what the other person really means, and then they stop paying attention to the other person.
+
+##### Cutting each other
+
+Only one person speaks!
+
+This is one reason why the meeting continues after the designated time and lasts for even an hour!
+
+##### Everyone addressing to one person
+
+The meeting is not a project status report meeting! It is a sync up call.
+Everyone should address the team, not the team lead or project manager.
+
+Non devs or attendees who are not directly working with the team should be quiet or give their updates to
+everyone like any other person but never be a receiver.
+
+##### Slip in other thing.
+
+"Let me show the demo of what I did", "Hey, if you are facing this issue, then do this to solve it", "Why don't you do
+this?", "Why don't we do this?".
+None of this should happen. Everyone should focus on only strictly updates. Nothing else.
+
+Did I say nothing else?
+
+If you want another conversation, meet after the meeting and have a separate meeting with a smaller group.
+Even if the next meeting will have the same participants, don't do it now; call another meeting and do it separately.
+
+##### Q: But we work only remotely, and this is the only time the whole team is together; I would want it to last a little longer and combine other things.
+
+Even in that case, keep the standup time shorter. Be on the same call with the same people, just complete standup in a short time.
+Don't slip in a demo along with standup. Do other things after officially concluding standup. Wear a different hat after that. 
+
+Remember, the standup should be completed on time in a short duration. 
+The meeting or call can be extended with the same or different participants.
+
+##### Too many update
+
+"I worked on setting up that server then talked to Bob then talked to Yoda about order detail then talked to Groot about
```

Comment:
May be add "number of updates or time you take to update others is not a measure of success or hardwork"